api/server.ts

```diff
@@ -48,11 +48,13 @@ handle404Error(app);
 handleClientError(app);
 handleServerError(app);
 
-app.listen(
-  PORT,
-  async (): Promise<void> => {
-    log.info(`Server is running on port ${PORT}...`);
-  }
-);
+if (!IS_TESTING) {
```

Comment:
I don't think we should have this check because eventually I wanted to have a test for this file and we would not be able to test against `app.listen()` if it doesn't get executed

Reply:
we don't want this to run during (most) tests though. It listens on a port and that's not cleaned up after the tests. This part could be split into a different file so we don't need the conditional, but I think we can cross that bridge when we get to it